### ISSUE TITLE
"help" as an alias for "?"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#49]
```
What steps will reproduce the problem?
1. typing "help" is an unknown command 

What is the expected output? What do you see instead?
"help" should be an alias for ? and produce the same output

What version of the product are you using? On what operating system?
GTalkSMS-v1.7-signed-aligned.apk 

Please provide any additional information below.


```

Original issue reported on code.google.com by `d...@dns.id.au` on 14 Jan 2011 at 2:08

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 14 Jan 2011 at 10:13
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 16 Jan 2011 at 11:54
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.7.2
```

Original comment by `Florent....@gmail.com` on 16 Jan 2011 at 7:44
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
